server/models/Cart.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require("mongoose");
```

Comment:
require("mongoose") should be require('mongoose') single quotes on every double quotes.

Reply:
ok thanks